File: 混合开发/Swift项目中调用Objective-C/Swift项目中调用Objective-C/Person.m
//
//  Person.m
//  Swift项目中调用Objective-C
//
//  Created by 杨帆 on 2019/7/18.
//  Copyright © 2019 杨帆. All rights reserved.
//

#import "Person.h"

@implementation Person
-(void)eat{
    NSString *result = [NSString stringWithFormat:@"%@吃饭了", self.name];
    NSLog(@"%@", result);
}
@end

File: 混合开发/Objective-C项目中调用Swift/Objective-C项目中调用Swift/Person.swift
//
//  Person.swift
//  Objective-C项目中调用Swift
//
//  Created by 杨帆 on 2019/7/18.
//  Copyright © 2019 杨帆. All rights reserved.
//

import UIKit

class Person: NSObject {
    
    //OC类能访问
    @objc var name:String?
    //OC类不能访问
    var age:Int?
    //OC类能访问
    @objc func eat(){
        print("\(name ?? "")吃饭了")
    }

}
